devicetypes/smartthings/zwave-metering-switch-secure.src/zwave-metering-switch-secure.groovy

```diff
@@ -0,0 +1,316 @@
+/**
+ *  Copyright 2015 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition (name: "Z-Wave Metering Switch Secure", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.switch") {
+		capability "Actuator"
+		capability "Configuration"
+		capability "Energy Meter"
+		capability "Health Check"
+		capability "Light"
+		capability "Polling"
+		capability "Power Meter"
+		capability "Refresh"
+		capability "Sensor"
+		capability "Switch"
+
+		command "reset"
+
+		fingerprint deviceId: "0x1001", inClusters: "0x5E, 0x22, 0x85, 0x59, 0x70, 0x56, 0x5A, 0x7A, 0x72, 0x32, 0x8E, 0x71, 0x73, 0x98, 0x31, 0x25, 0x86", outClusters: ""
+		fingerprint mfr:"0072", prod:"0501", model:"0F06", deviceJoinName: "FIBARO Wall Plug ZW5"
+	}
+
+	// simulator metadata
+	simulator {
+		status "on":  "command: 2003, payload: FF"
+		status "off": "command: 2003, payload: 00"
+
+		for (int i = 0; i <= 10000; i += 1000) {
+			status "power  ${i} W": new physicalgraph.zwave.Zwave().meterV1.meterReport(
+					scaledMeterValue: i, precision: 3, meterType: 4, scale: 2, size: 4).incomingMessage()
+		}
+		for (int i = 0; i <= 100; i += 10) {
+			status "energy  ${i} kWh": new physicalgraph.zwave.Zwave().meterV1.meterReport(
+					scaledMeterValue: i, precision: 3, meterType: 0, scale: 0, size: 4).incomingMessage()
+		}
+
+		// reply messages
+		reply "2001FF,delay 100,2502": "command: 2503, payload: FF"
+		reply "200100,delay 100,2502": "command: 2503, payload: 00"
+	}
+
+	tiles {
```

Comment:
@parijatdas Please use multiAttribute tiles to get 2x2 look. 
![201708080025720_pymc4cbb](https://user-images.githubusercontent.com/255235/29042369-890e9602-7b6b-11e7-8c4c-7744594d5212.jpg) that's the older view